src/app/components/users-page/users-page.component.html

```diff
@@ -1,13 +1,18 @@
 <br>
 <h1 class="title"> LISTE DES UTILISATEURS </h1>
-<span class="custom-br"></span>
-<br>
-<span class="d-block p-2 bg-dark text-white user-title"> Matbay Staff </span>
+<hr>
 <br>
 <div class="container">
-	<div class="row g-4">
+	<div class="admin-color blob">
+		<h2 class="user-title">Équipe Technique
```

Comment:
Proposition pour uniformiser les titres de sections. Également, est-ce que c'est normal qu'il y a un espace plus important entre les deux mots ?
```suggestion
		<h2 class="user-title">Équipe Matbay
```

Reply:
Alors oui c'est normal mais pour être honnête je ne me rappelle plus de pourquoi j'avais fait cette modification, je crois que c'était par rapport à l'svg qui devait avoir une certaine largeur ou quelque chose comme ça, mais on peut toujours supprimer `word-spacing: 15px;` dans `src/app/components/users-page/users-page.component.scss
`
```scss
.user-title{
    margin: auto auto 10px;
    text-align: center;
    font-size: 30px;
    font-weight: 100;
    / word-spacing: 15px;
    width: max-content;
}
```